css/style.css

```diff
@@ -0,0 +1,100 @@
+
+
+body {
+	font-family: Arial;
+	background: black;
+  -webkit-font-smoothing: antialiased;
+}
+
+hgroup { 
+	text-align:center;
+	margin-top:0;
+    color: black;
+}
+
+h1, h3 { font-weight: 300; }
+
+h1 { color: gainsboro; }
+
+h3 { color: #4a89dc; }
+
+form {
+	width: 380px;
+	margin: 7em auto;
+	padding: 3em 2em 2em 2em;
+	background: #fafafa;
+	border: 1px solid #ebebeb;
+	box-shadow: rgba(0,0,0,0.14902) 0px 1px 1px 0px,rgba(0,0,0,0.09804) 0px 1px 2px 0px;
```

Comment:
Very interesting values for the alpha channel! Care yo elaborate on how you chose them?